modules/electrophysiology_browser/help/view_session.md

```diff
@@ -0,0 +1,9 @@
+# View Session
+
+This page displays all available information collected for an individual scan session.
```

Comment:
```suggestion
This page displays all available information collected for an individual recording session.
```
because eeg tends to use the word recording.  Scan is more MRI.